### PR TITLE
Another proposal for notifications.

### PR DESCRIPTION
**Pull request by TheBox193**
Simple variant of desktop notifications.
* Notification only if count increases
* Notification contains count of unread
* Notification links to GitHub Notifications

image

Pulled Settings portion in from https://github.com/sindresorhus/github-notifier-chrome/pull/25

### CODE REVIEW

File: extension/main.js [hunk 1]
```diff
@@ -15,6 +15,61 @@
...
+				openGithub();
+			};
+		} else if (Notification.permission !== 'denied') {
+			// Otherwise, we need to ask the user for permission
+			Notification.requestPermission();
+		}
+
+		// At last, if the user already denied any notification, and you
+		// want to be respectful there is no need to bother them any more.
+	}
```
Comment (sindresorhus) [inline extension/main.js]: Should use the extension notification API; https://developer.chrome.com/apps/notifications

**Comment by sindresorhus** [comment]
Yeah, this is better than nothing, though I still think it would be a lot more useful if it had desktop notifications for individual GitHub notifications. Let's go with this for now though. Thanks :)

**Comment by sindresorhus** [comment]
@TheBox193 Interested in finishing this?

**Comment by TheBox193** [comment]
@sindresorhus Yeah, when I get a moment. It's on my mind still.

**Comment by sindresorhus** [comment]
Closing for now. Happy to reopen again if you ever get to this @TheBox193 ;)

**Comment by TheBox193** [comment]
Sure thing :+1:

**Pull request closed without merging**